db/dev-or-prod-migrations.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="replace-dots-with-hyphens" author="sanyavertolet" context="">
+        <sql>
+            update save_cloud.`user` set name = replace (name, '.', '-');
+            update save_cloud.project set name = replace (name, '.', '-');
+            update save_cloud.organization set name = replace (name, '.', '-');
+            update save_cloud.contest set name = replace (name, '.', '-');
+        </sql>
+    </changeSet>
+
+    <changeSet id="insert-high-level-names-data" author="sanyavertolet" context="">
```

Comment:
You have already created such a changeSet in _high-level-names.xal_

Reply:
Lol, it should be in `dev-or-prod-migrations.xml` because we also want inserted test data processed on dev

Reply:
Fixed